### DISCUSSION TITLE
Bug in VersionInfo.cpp

### DISCUSSION THREAD

**Discussion by he-igel** [ImageMagick/ImageMagick#5047] [General] [answered]
Hello community,

i think there is a bug in ImageMagick\VisualMagick\configure\VersionInfo.cpp :

Code :

void VersionInfo::setReleaseDate()
{
  _releaseDate=executeCommand(L"cd ..\\..\\ImageMagick && git log -1 --format=%cd --date=format:%Y-%m-%d");
  if (**_gitRevision** == L"")
    **_gitRevision**=getFileModificationDate(L"..\\..\\ImageMagick\\ChangeLog.md",L"%Y-%m-%d");
}

but it shoud be :

void VersionInfo::setReleaseDate()
{
  _releaseDate=executeCommand(L"cd ..\\..\\ImageMagick && git log -1 --format=%cd --date=format:%Y-%m-%d");
  if (**_releaseDate** == L"")
    **_releaseDate**=getFileModificationDate(L"..\\..\\ImageMagick\\ChangeLog.md",L"%Y-%m-%d");
}

Regards

**Comment by dlemstra** [comment, marked as answer]
Thanks for reporting this. Made a patch for this and pushed this to the repository. Feel free to open up a pull request for this next time you find something.
